docs/using-the-nodejs-wrapper/DriverDialects.md

```diff
@@ -0,0 +1,14 @@
+# Driver Dialects
+
+## What are Driver Dialects?
+
+The AWS Advanced NodeJS Wrapper is a wrapper that requires an underlying driver, and it is meant to be compatible with any NodeJS driver. Driver dialects help the AWS Advanced NodeJS Wrapper to properly pass calls to a target driver. To function correctly, the AWS Advanced NodeJS Wrapper requires details unique to the specific target driver such as the method to create the current connection, whether to include some specific connection parameters, etc. These details can be defined and provided to the AWS Advanced NodeJS Wrapper by using driver dialects.
```

Comment:
It is only compatible with 2 drivers right now, lets change `and it is meant to be compatible with any NodeJS driver` to `it is currently compatible with pg (link) and mysql2 (link).